zeebe/broker-client/src/test/java/io/camunda/zeebe/broker/client/api/BrokerTopologyManagerTest.java

```diff
@@ -363,6 +367,28 @@ void shouldNotifyListenerWhenBrokerRemoved() {
     assertThat(topology.getBrokers()).doesNotContain(brokerId);
   }
 
+  @Test
+  void shouldNotifyListenerWhenClusterChangeCompleted() {
+    // given
+    final ClusterCompletedChangeListener listener = new ClusterCompletedChangeListener();
+    addTopologyListener(listener);
+
+    // when
+    final ClusterConfiguration clusterTopologyWithTwoBrokers =
```

Comment:
:upside_down_face: probably the name is just copied from somewhere but what's relevant here is that the cluster topology has a completed change, not that it has two brokers.